src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const payment = 40;
+
+  if (days >= 7) {
+    return days * payment - 50;
+  }
+
+  if (days >= 3 && days <= 7) {
```

Comment:
The condition `days >= 3 && days <= 7` is incorrect because it overlaps with the condition `days >= 7` from the previous if statement. The correct condition should be `days >= 3 && days < 7` to ensure that the discount logic is applied correctly.